examples/sampling/graphbolt/link_prediction.py

```diff
@@ -205,7 +207,7 @@ def create_dataloader(args, graph, features, itemset, is_train=True):
     return dataloader
 
 
-def to_binary_link_dgl_computing_pack(data: gb.MiniBatch):
+def to_binary_link_dgl_computing_pack(data: gb.DGLMiniBatch):
```

Comment:
Is this just a helper function or could we make it a member function of `gb.DGLMiniBatch` ?

Reply:
Since the `datapipe` do not generate corresponding label, we need to construct them manually. But, I do think it will be merged into the `datapipe` in the future.